src/Controllers/ElementalAreaController.php

```diff
@@ -45,132 +47,158 @@ public function getClientConfig()
         $clientConfig = parent::getClientConfig();
         $clientConfig['form']['elementForm'] = [
             'schemaUrl' => $this->Link('schema/elementForm'),
-            'saveUrl' => $this->Link('api/saveForm'),
-            'saveMethod' => 'post',
-            'payloadFormat' => 'json',
-            'formNameTemplate' => sprintf(static::FORM_NAME_TEMPLATE ?? '', '{id}'),
+            'formNameTemplate' => sprintf(static::FORM_NAME_TEMPLATE, '{id}'),
         ];
-
         // Configuration that is available per element type
         $clientConfig['elementTypes'] = ElementTypeRegistry::generate()->getDefinitions();
-
         return $clientConfig;
     }
 
     /**
-     * @param HTTPRequest|null $request
+     * Used for both:
+     * - GET requests to get the FormSchema via getElementForm() called from LeftAndMain::schema()
+     * - POST Requests to save the Form. Will be handled by to FormRequestHandler::httpSubmission()
+     * /admin/linkfield/elementForm/<ElementID>
+     *
+     * @param HTTPRequest $request - deprecated 5.3.0 Will be removed without equivalent functionality to replace it
      * @return Form
-     * @throws HTTPResponse_Exception
      */
     public function elementForm(HTTPRequest $request = null)
     {
-        // Get ID either from posted back value, or url parameter
-        if (!$request) {
-            $this->jsonError(400);
-            return null;
-        }
-        $id = $request->param('ID');
+        $request = $request ?: $this->getRequest();
+        $id = $request->param('ItemID');
```

Comment:
I think ID used to work with the old routing because it's a one of the default params from `/$Action/$ID/$OtherID`

$ItemID is correct for both of the code paths mentioned in the docblock comment